kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/PppConfigWriter.java

```diff
@@ -192,6 +218,10 @@ public String formPeerFilename(UsbDevice usbDevice) {
     }
 
     public String formPppLogFilename(UsbDevice usbDevice) {
+        if (!this.logEnabled) {
+            return null;
+        }
+        
```

Comment:
If the logEnabled is false, the filename is not written in the peers file. But, where will be written the logs?

Reply:
PPP can write directly in the journal for systemd systems